### PR TITLE
fix:code is for uploading feedback images

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

This code is for uploading feedback images  to blob

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/modules/workspace/controllers/feedback.controller.ts [hunk 1]
```diff
@@ -109,4 +109,49 @@ export class FeedbackController {
...
+  @ApiOperation({
+    summary: "Add a Feedback",
+    description: "You can add a Feedback",
+  }) // Provides metadata for this operation in Swagger documentation
+  @ApiConsumes("multipart/form-data") // Specifies that this endpoint consumes multipart/form-data
+  @ApiBody({
+    schema: {
+      type: "object",
+      properties: {
+        files: {
```
Comment (itsmdasifraza) [inline src/modules/workspace/controllers/feedback.controller.ts:128]: update comments

File: src/modules/workspace/controllers/feedback.controller.ts [hunk 2]
```diff
@@ -109,4 +109,49 @@ export class FeedbackController {
...
+  async uploadFeedbackFile(
+    @Res() res: FastifyReply,
+    @UploadedFiles()
+    files: MemoryStorageFile[],
+  ) {
+    const uploads = await this.feedbackService.uploadFeedbackFile(files); // Calls the feedback service to add new feedback
+    console.log(uploads, "uploads");
+
+    const responseData = new ApiResponseService(
+      "Feedback Added",
```
Comment (itsmdasifraza) [inline src/modules/workspace/controllers/feedback.controller.ts:151]: remove consoles

File: src/modules/workspace/services/feedback.service.ts [hunk 3]
```diff
@@ -104,4 +104,15 @@ export class FeedbackService {
     );
     return response;
   }
+
+  async uploadFeedbackFile(files: MemoryStorageFile[]) {
+    // Validate the uploaded files
+    await this.isFeedbackFilesValid(files);
+
+    // Upload all files to the blob storage service in parallel
```
Comment (itsmdasifraza) [inline src/modules/workspace/services/feedback.service.ts:112]: update media upload validation

**Review by itsmdasifraza — APPROVED** [review]

**Review by itsmdasifraza — APPROVED** [review]

File: src/modules/workspace/services/feedback.service.ts [hunk 4]
```diff
@@ -104,4 +104,15 @@ export class FeedbackService {
     );
     return response;
   }
+
+  async uploadFeedbackFile(files: MemoryStorageFile[]) {
```
Comment (Astitva877) [inline src/modules/workspace/services/feedback.service.ts:108]: add return type here.

**Pull request merged**